go.mod

```diff
@@ -6,6 +6,7 @@ go 1.18
 replace maze.io/x/crypto => github.com/snapcore/maze.io-x-crypto v0.0.0-20190131090603-9b94c9afe066
 
 require (
+	github.com/bmatcuk/doublestar/v4 v4.6.1
```

Comment:
I see some dependencies in packaging/fedora/snapd.spec, for example, but this isn't something I've worked on before. Is this a build dependency (I would think, since Go statically compiles?) or runtime dependency (but then, this might be why we need the dependency as a package?)?